.travis.yml

```diff
@@ -5,6 +5,7 @@ go:
 
 before_install:
   - go get -v
+  - go fmt
```

Comment:
`go fmt .` (notice the dot) will list the files that have unformatted content. With that you can check if the code is formatted properly by checking if the output is empty.

If only we had a static code analysis tool that does all this automatically ;) Wanna add a coafile instead?

Reply:
If it is supported why not.